src/components/index.js

```diff
@@ -124,13 +124,13 @@ export {default as CloudTile} from './modals/cloud-tile';
 export {default as FileUploadFactory, FileUpload} from './common/file-uploader/file-upload';
 export {default as DatasetLabel} from './common/dataset-label';
 export {default as ItemSelector} from './common/item-selector/item-selector';
-export {default as FieldSelector} from './common/field-selector';
```

Comment:
this will break the app that is importing `FieldSelector `
take a look at how `TimeRangeSliderFactory` and `TimeRangeSlider` are exported and do this same

Reply:
Fixed. nice catch, feel free to take a look